pkgs/development/compilers/graalvm/graalvm-oracle/hashes.nix

```diff
@@ -0,0 +1,42 @@
+{
+  "22" = {
+    "hashes" = {
+      "aarch64-linux" = {
+        sha256 = "d303d30c10764feaab076efe79019df3c981b0487fcac3df7a5313f76f61040e";
+        url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_linux-aarch64_bin.tar.gz";
+      };
+      "x86_64-linux" = {
+        sha256 = "d1860b5b7588310e70b259c891156f6d0cbc34d0d1feec3b37169ed2a415f3c3";
+        url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_linux-x64_bin.tar.gz";
+      };
+      "x86_64-darwin" = {
+        sha256 = "5b83f20dbc4c636ed41f19c3309f09839d4f5c6442dba986f460589c494a476c";
+        url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_macos-x64_bin.tar.gz";
+      };
+      "aarch64-darwin" = {
+        sha256 = "61632065cfcdc4e121362f1fd25a543955836bbacd6c1aadbcbe0d469d5ab8a3";
+        url = "https://download.oracle.com/graalvm/22/archive/graalvm-jdk-22_macos-aarch64_bin.tar.gz";
+      };
+    };
+  };
+  "17" = {
+    "hashes" = {
+      "aarch64-linux" = {
+        sha256 = "1s3ma7x22wqg10cqhdgwj0g9qk3n7bqy10dq6x1vgpljdd41v6dx";
+        url = "https://download.oracle.com/graalvm/17/latest/graalvm-jdk-17_linux-aarch64_bin.tar.gz";
```

Comment:
Not sure if this URL is deterministic, there is no `/archive/` version for these